### PR TITLE
Database layer and start of business models

### PR DESCRIPTION
**Pull request by MariusBBD**
Database layer is completed. I renamed the database models to entities so that when we have DTO's or business models that the naming don't clash.

I added the first business model Commuter. Please let me know if that is fine with you.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+.idea
```
Comment (HylaBBD) [inline .gitignore:25]: I'm not familiar with this type of file. What generates a .idea file?

File: backend/src/main/java/com/commutetrip/backend/database/entities/TruckRoutesEntity.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
 @NoArgsConstructor
 @Entity
 @Table(name = "truck_routes")
-public class TruckRoutes {
+public class TruckRoutesEntity {
```
Comment (HylaBBD) [inline backend/src/main/java/com/commutetrip/backend/database/entities/TruckRoutesEntity.java:20]: Ahhh I should have called this TruckRoute not Routes

File: .gitignore (hunk 1 above)
Comment (MariusBBD) [reply .gitignore:25]: IntelliJ IDEA generates it

File: backend/src/main/java/com/commutetrip/backend/database/entities/TruckRoutesEntity.java (hunk 2 above)
Comment (MariusBBD) [reply backend/src/main/java/com/commutetrip/backend/database/entities/TruckRoutesEntity.java:20]: We can make the changes if you want

Comment (HylaBBD) [reply backend/src/main/java/com/commutetrip/backend/database/entities/TruckRoutesEntity.java:20]: please do :)

Comment (MariusBBD) [reply backend/src/main/java/com/commutetrip/backend/database/entities/TruckRoutesEntity.java:20]: Ok will make the changes and push it up

**Review by HylaBBD — APPROVED** [review]

**Pull request merged**